person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initilialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Kindly make sure that of_age is a private method as per the requirements.

For this add the of_age method after can_use services so that only of_age is private.

Reply:
ok fixed